### PR TITLE
doc(kubectl): Kubectl restart command without specific deployment wil…

### PR DESCRIPTION
**Pull request by Ithrael**
…l restart all deployments

#### What type of PR is this?

/kind documentation

#### What this PR does / why we need it:
Kubectl restart command without specific deployment will restart all deployments, Add an example in the help text to mention this behavior.

#### Which issue(s) this PR fixes:

Fixes # https://github.com/kubernetes/kubectl/issues/1474

#### Does this PR introduce a user-facing change?

```release-note
Added descriptions and examples for the situation of using kubectl rollout restart without specifying a particular deployment.
```

#### Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.:

```docs

```

### CODE REVIEW

File: staging/src/k8s.io/kubectl/pkg/cmd/rollout/rollout_restart.go [hunk 1]
```diff
@@ -63,6 +63,9 @@ var (
 	        Resource rollout will be restarted.`))
 
 	restartExample = templates.Examples(`
+		# Restart all deployments in default namespace
+		kubectl rollout restart deployment
```
Comment (ardaguclu) [inline staging/src/k8s.io/kubectl/pkg/cmd/rollout/rollout_restart.go:67]: nit: would be better to add an example by explicitly specifying a namespace such as `kubectl rollout restart deployment -n test-namespace`

**Review by ardaguclu — COMMENTED** [review]
/approve
/lgtm

**Comment by sftim** [comment]
Is this change visible to end users? If so, we could add a changelog entry.

**Comment by Ithrael** [comment]
> Is this change visible to end users? If so, we could add a changelog entry.

Yes, when users execute `kubectl rollout restart -h`, the examples they see will change.

**Comment by sftim** [comment]
Please set a changelog for this @Ithrael (you do that by editing the PR description, there are also helper Prow commands).

**Pull request merged**